thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js

```diff
@@ -127,8 +135,8 @@ export default Ember.Component.extend({
     'selectedUrns',
     'analysisRange',
```

Comment:
does this still need to depend on 'analysisRange' ?

Reply:
fixed